src/BackgroundTasks/Task.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace ILIAS\BackgroundTasks;
+
+use ILIAS\Types\Type;
+
+/**
+ * Interface Task
+ *
+ * @package ILIAS\BackgroundTasks
+ *
+ *          A Task is the basic interface of an "thing" which can be put into a Bucket and will be
+ *          run or triggered by the BackgroundTask-Worker. Currently there are two types of Tasks:
+ *          - Job: A Task, which can be run without any interaction with the user such as zipping
+ *          files or just collecting some data
+ *          - UserInteraction: A Task in the Bucket, which will need some User-Interaction before
+ *          running the task. A User-Interaction is provided as Button in the UserInterface such as
+ *          [ Cancel ] or [ Download ]
+ */
+interface Task {
+
+	/**
+	 * @return string
```

Comment:
Ok, I guess it should not be `Type`, because we also have `getOutputType`. What is this, then? Is this the class of the `Task`? Why would I want to use this, then, instead of `get_class`?